site/cypress/e2e/Cypress-UI/pronouns_change.spec.js

```diff
@@ -36,16 +44,34 @@ describe('Tests cases abut changing user pronouns', () => {
 
         //change back to old pronouns
         cy.get('#pronouns_val').click();
+
         cy.get('button[aria-label="Clear pronoun input"]').click(); //clear input using trash can
         if (oldPronouns !== '') {
             cy.get('#user-pronouns-change').type(oldPronouns);
         }
-        cy.get('#edit-pronouns-submit').first().click();
 
-        //ensure pronouns changed on page
+        //change display option back
+        cy.get('#pronouns-forum-display').then(() => {
```

Comment:
if this is only running with the student profile, would it be more concise to hard-code the resetting of the variables, because you should be able to see what the original values are ?

Reply:
yea, but the variables might be different or changed by different users. So In the future, if people run the cypress test locally. Their student account's variable will be changed to the original values I set, instead of their original value.

Reply:
Will they be changing the student account pronoun for a reason on a dev VM? (besides to test something)

Reply:
I think it makes sense to try to accept a system in any state, and then restore the system to the initial state when the test is done whenever possible.  That said, we do have a fair number of tests which rely upon the system being in a fresh state, so it's not terrible to do it that way either...